src/meta-srv/src/procedure/region_migration.rs

```diff
@@ -80,6 +83,29 @@ pub struct VolatileContext {
     opening_region_guard: Option<OpeningRegionGuard>,
     /// `table_route_info` is stored via previous steps for future use.
     table_route_info: Option<DeserializedValueWithBytes<TableRouteValue>>,
+    /// The deadline of leader region lease.
+    leader_region_lease_deadline: Option<Instant>,
```

Comment:
What's the actual usage of this field other than some assertions in the tests?

Reply:
If all open candidate region retries are failed, It will wait to exceed the deadline.